ush/python/pygfs/task/atm_analysis.py

```diff
@@ -103,7 +103,7 @@ def initialize(self: Analysis) -> None:
 
         # generate variational YAML file
         logger.debug(f"Generate variational YAML file: {self.task_config.fv3jedi_yaml}")
-        varda_yaml = parse_j2yaml(self.task_config.ATMVARYAML, self.task_config)
+        varda_yaml = parse_j2yaml(self.task_config.ATMVARYAML, self.task_config, searchpath=self.gdasapp_j2tmpl_dir)
```

Comment:
```suggestion
        varda_yaml = parse_j2yaml(self.task_config.JEDIYAML, self.task_config, searchpath=self.gdasapp_j2tmpl_dir)
```
replace `ATMVARYAML` with `JEDIYAML` here if this variable is renamed in `config.atmanl`

Reply:
Change committed at ed042392.